examples/vanilla-ts-esm/public/mux-viewer-count.html

```diff
@@ -0,0 +1,21 @@
+<!-- eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJwbGF5YmFja19pZCIsImV4cCI6MTY3NDg2NjU4NCwia2lkIjoiWkV5TlA2OWY1Z1RQQ1V2MDJ5N0FNRjAwQ2hwRUFZS1ViWThpOXk5ekNnQWVZIiwic3ViIjoiWWtnSFdTYTVDT0NYWWJuV2ZrR1loZzRvNm9CTk5QWFBHNzAxQ2tjY2RkV2MifQ.SwJR4G8u-kI9pFQ9r4MwZ06bDiZyFyQGHYO-Tg89W3ZT6L6dUQ6xQ_PhRFlNfSqc3rIooP5aBBDXWQfDBBgUPcxkZ_SQPWqZFvhdNyuT_vEeOpJXE_Sg7DcpAq5pXkm5H8HR5mjsslHoXEN1-6495ReKiiCW6h1GEpzo1O9RkanLybONttiXJW-IK59EvlB6QxYrxsr1hslLJd1z1SMyYhGxTEpw0IeXzWkLm49wI-MIDN87fb14ydiPtJ2pli5cmp2jOdoQGlQa5eFtavVuB-7zGQiSPoJ8fXGQruNdPtsZ34Z25lkRu55hed_adDK0BF84mSqZxhrWCmBuHQGmZg -->
+<!DOCTYPE html>
+<html>
+
+<head>
+  <style>
+    media-viewer-count {
+      display: inline-block;
+      color: blue;
+    }
+
+  </style>
+  <meta charset="UTF-8">
+</head>
+
+<body>
+  <media-viewer-count interval="10" token="eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJwbGF5YmFja19pZCIsImV4cCI6MTY3NDkzODcxMCwia2lkIjoiWkV5TlA2OWY1Z1RQQ1V2MDJ5N0FNRjAwQ2hwRUFZS1ViWThpOXk5ekNnQWVZIiwic3ViIjoiWWtnSFdTYTVDT0NYWWJuV2ZrR1loZzRvNm9CTk5QWFBHNzAxQ2tjY2RkV2MifQ.H5baSYnpdQNzhbiNwHX2v2z6cdwiskWhQmWvCXBE7fiwKeM8vmGHoHQ1bvajGCA93qBpyhJkBFEbdMahNCGzGQs_EABsoJ7jnWMsYY874WWKukrYWJHJUTZp3bLTx00SiDLcr5Nky_KZTYAQ2diiy6vpIfNtwzrTX1C7Lyfn38h39HlLXrbKNMpP8XUog6_zz4_EDcT60S4QRB63AnWwP-mRFh97F4ujzH2sZ9C8QIpcncH5AYTIwV0i-pHSGdzyw9KYxG6l2-uu44nBDootOZzsYrxW-DVWgON4r_QM2EG0ej79rLnOqL2el6PbRpLbgrQ-SJ1zMFscqUX_rXvcqA"></media-viewer-count>
+  <script type='module' src='./dist/mux-viewer-count.js'></script>
```

Comment:
Do we want to actually put this token in here, or just a placeholder?

Reply:
Nah we'll refactor this so a user can add it via an input. That will require supporting "dynamic updates" via e.g. `attributeChangedCallback`, which still needs to be added to the implementation.